internal/integration/bounce_rule_test.go

```diff
@@ -102,15 +111,18 @@ func (suite *BounceRuleSuite) TestPostBounceRuleHandler() {
 
 	assert.Equal(suite.T(), http.StatusNotFound, resp.StatusCode)
 
-	reqBody := map[string]interface{}{
-		"ResponseCode": 421,
-		"EnhancedCode": "5235123",
-		"Regex":        "1111111",
-		"Priority":     0,
-		"Description":  "RFC5321 Service not available",
-		"BounceAction": "retry",
+	have := models.BounceRule{
```

Comment:
These statements are backwards :(

`want` is what you're looking for, `have` is what you got. So your `want` is what you expect to receive from the server, and your `have` will be what you actually receive back

Reply:
oh haha, yeah a little confusing.

I'm going to wait until Gev's PR gets merged in because I think one of his updates affects one of the tests so might as well wait for it to go through first